README.old.md

```diff
@@ -0,0 +1,68 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+
+> Description the project.
+
+
+## Built With
+
+- Major languages
+- Frameworks
+- Technologies used
```

Comment:
- [ ] It will be nice to have a list of all technologies used for your app in the  Built With 🛠️ section 👍🏻